pkgs/development/python-modules/dropbox/default.nix

```diff
@@ -11,16 +11,16 @@
 
 buildPythonPackage rec {
   pname = "dropbox";
-  version = "11.29.0";
+  version = "11.30.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "dropbox";
     repo = "dropbox-sdk-python";
-    rev = "v${version}";
-    sha256 = "sha256-TKJb34hJYzZtQcqgopLpN8c1utWCNjmev9epY+hYU7M=";
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
https://github.com/NixOS/nixpkgs/pull/169285#discussion_r853029123

I did saw this exactly behavior in a package that I maintain (`uasm`), so maybe it is better to keep the old way.